beetmoverscript/utils.py

```diff
@@ -309,6 +309,14 @@ def validated_task_id(task_id):
     raise ValueError("No valid taskId found.")
 
 
+def exists_or_endswith(filename, list_of_artifacts):
+    for artifact in list_of_artifacts:
+        if filename == artifact or filename.endswith(artifact):
```

Comment:
This function gets given `BUILDHUB_ARTIFACT`, which is a string, but it's treating it as a list. So `for artifact in list_of_artifacts` iterates over `['b', 'u', 'i', 'l', 'd', 'h', 'u', 'b', '.', 'j', 's', 'o', 'n']` and will return true for any filename which ends with any of those letters. I think this is why the CI test is failing with json load errors - it's been told to load files that aren't json.

Reply:
Sorry, I forgot to push my local changes, I had fixed that in the meantime.